README.md

```diff
@@ -17,7 +17,6 @@ Authors  :: Vikash Kumar (vikashplus@gmail.com), Vittorio Caggiano (caggiano@gma
 `MyoSuite` is a collection of musculoskeletal environments and tasks simulated with the [MuJoCo](http://www.mujoco.org/) physics engine and wrapped in the OpenAI ``gym`` API to enable the application of Machine Learning to bio-mechanic control problems.
 
 
- <!-- TODO: add brieft history of the mujoco environment? -->
 <!-- TODO: the tutorials section needs to format as now it is mixing all versions -->
```

Comment:
@raku-slyu @Vittorio-Caggiano 
What's our plan here?

Reply:
This was inspired from the mujoco and stable_baseline repo regarding a brief introduction of the environment and key features of why using myosuite (potentially make the term contact-rich dynamics more prominent). But removed it as the myosuite page is quite self-contained

Reply:
removed and possibly include in next change